src/entity.ts

```diff
@@ -218,7 +227,7 @@ export class Entity {
 				else if (typeof state !== "object")
 					value = state;
 				else if (currentValue && !getIdFromState(ChildEntity.meta, state))
-					currentValue.withContext(this._context, entity => entity.set(state));
+					currentValue.updateInContext(this._context, state);
```

Comment:
Is this a typo?

Reply:
Yep. Intellisense must not have auto-renamed that occurrence. Thanks.